model/converter/thrift/jaeger/to_domain.go

```diff
@@ -74,9 +74,17 @@ func (td toDomain) transformSpan(jSpan *jaeger.Span, mProcess *model.Process) *m
 		Tags:          tags,
 		Logs:          td.getLogs(jSpan.Logs),
 		Process:       mProcess,
+		Incomplete:    td.checkIncomplete(jSpan.Incomplete),
 	}
 }
 
+func (td toDomain) checkIncomplete(incomplete *bool) bool {
+	if incomplete == nil {
+		return false
+	}
+	return *incomplete
```

Comment:
nit: could be simplified to `return incomplete != nil && *incomplete`

Reply:
Will do that.